CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Replace moment with day.js. Refs UIREQ-1291.
 * Reduce count of eslint errors after update eslint-config-stripes. Refs UIREQ-1289.
 * Display loan type and "For use at location" loan-policy setting in new-request form. Fixes UIREQ-1282.
+* When a patron record is deleted there is still a link in the requests details page. Refs UIREQ-1312.
```

Comment:
I guess it is better to describe the change the way "what has been done", instead of just describing a problem. Like "Display Unknown user instead of a link when a patron record is deleted"

Reply:
Fixed